File: 0001 Code Chef/0003 Good Turn/implementation/0009 dart/good_turn.dart
import 'dart:io';

void main(){
  String? stringInput = stdin.readLineSync();
  if(stringInput != null){
    List<String> stringArray = stringInput.split(" ");
    int? x = int.tryParse(stringArray[0]);
    int? y = int.tryParse(stringArray[1]);
    if(x != null && y != null){
      int sumof = x + y;
      if(sumof > 6){
        print("YES");
      }else{
        print("NO");
      }
    }


  }
}

File: 0001 Code Chef/0003 Good Turn/implementation/0008 python3/good_turn.py
if __name__ == "__main__":
    string_input = input();
    string_array = string_input.split(" ")
    x = int(string_array[0]);
    y = int(string_array[1]);
    sumof = x + y
    if(sumof > 6):
        print("YES")
    else:
        print("NO")